docs/streams/upgrade-guide.html

```diff
@@ -99,6 +100,14 @@ <h3><a id="streams_api_changes_210" href="#streams_api_changes_210">Streams API
         <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-321%3A+Update+TopologyDescription+to+better+represent+Source+and+Sink+Nodes">KIP-321</a>.
     </p>
 
+    <p>
+        We've added a new config named <code>max.task.idle.ms</code> to allow users specify how to handle out-of-ordering data within a task that may be processing multiple
+        topic-partitions (see <a href="/{{version}}/documentation/streams/core-concepts.html#streams_out_of_ordering">Out-of-Order Handling</a> section for more details).
+        The default value is set to <code>0</code>, to favor minimized latency over synchronization between multiple input streams from topic-partitions.
+        If users would like to wait for longer time when some of the topic-partitions are not data available to process and hence cannot determine its corresponding stream time,
```

Comment:
`topic-partitions are not data available` -> `topic-partitions [have no] data available` ?